PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterHadronicCorrection.cxx

```diff
@@ -640,15 +645,10 @@ void AliEmcalCorrectionClusterHadronicCorrection::DoMatchedTracksLoop(Int_t iclu
     //Do momentum dependent track matching. Values taken from the PCM analyses see:
     //https://alice-notes.web.cern.ch/node/411  Fig. 46   for mom<3GeV these cuts are wider than the standard cuts
```

Comment:
Can you please add these comments to where the functions are defined? I think they're more useful there than they are here (or maybe just keep them at both?)

Reply:
Done - thank you